tests/integ/test_git.py

```diff
@@ -60,7 +61,7 @@ def test_git_support_with_pytorch(sagemaker_local_session):
         entry_point=script_path,
         role="SageMakerRole",
         source_dir="pytorch",
-        framework_version=PyTorch.LATEST_VERSION,
+        framework_version=PYTORCH_VERSION,
```

Comment:
is this intentional?

Reply:
Yes. Because the script used in the test GitHub repo is not updated, and will fail when PyTorch version is 1.1.